default/python/universe_generation/monsters.py

```diff
@@ -33,7 +107,8 @@ def generate_monsters(monster_freq, systems):
                     if fp.spawn_rate() > 0.0 and fp.spawn_limit() > 0 and fp.ship_designs()}
 
     # map nests to monsters for ease of reporting
-    nest_name_map = dict(zip(["KRAKEN_NEST_SPECIAL", "SNOWFLAKE_NEST_SPECIAL", "JUGGERNAUT_NEST_SPECIAL"], ["SM_KRAKEN_1", "SM_SNOWFLAKE_1", "SM_JUGGERNAUT_1"]))
+    nest_name_map = dict(zip(["KRAKEN_NEST_SPECIAL", "SNOWFLAKE_NEST_SPECIAL", "JUGGERNAUT_NEST_SPECIAL"],
```

Comment:
Cleanup?

Merge it to master and rebase.   

PS. convert them to list of pairs to avoid zip